ui/raidboss/data/07-dt/raid/r1s.ts

```diff
@@ -36,6 +273,223 @@ const triggerSet: TriggerSet<Data> = {
+        if (actorSetPosLine === undefined)
+          return false;
+        const x = parseFloat(actorSetPosLine.x);
+        const y = parseFloat(actorSetPosLine.y);
+
+        const loc = Object.values(mapEffectData)
+          .find((tile) =>
+            tile.location.startsWith('0') && Math.abs(tile.centerX - x) < 1 &&
+            Math.abs(tile.centerY - y) < 1
+          );
+        if (loc === undefined)
+          return false;
+
+        const tile = loc.location;
+
+        if (tile !== '09' && tile !== '0A')
+          return false;
+
+        data.mouserMatchedTile = tile;
+        return true;
+      },
+      // We don't need a suppressSeconds since only one of the SW/SE tiles will get hit twice
+      durationSeconds: 11,
+      infoText: (data, _matches, output) => {
+        const dangerTile = data.mouserMatchedTile;
+        if (dangerTile === undefined)
+          return false;
+
+        // Danger tile is SW, so safe movement is SW => SE (Stay)
+        if (dangerTile === '09') {
+          return output.swSeStay!({
+            dir1: output['dirSW']!(),
+            sep: output.separator!(),
+            dir2: output['dirSE']!(),
+          });
+        }
+
+        const dirs = ['dirSW', 'dirSE', 'dirSW'].map((e) => output[e]!());
+
+        return output.combo!({ dirs: dirs.join(output.separator!()) });
+      },
+      run: (data) => delete data.mouserMatchedTile,
+      outputStrings: {
+        ...Directions.outputStrings8Dir,
+        swSeStay: {
+          en: '${dir1} ${sep} ${dir2} (Stay)',
+        },
+        separator: {
+          en: ' => ',
+          de: ' => ',
+          ja: ' => ',
+          cn: ' => ',
+        },
+        combo: {
+          en: '${dirs}',
+          de: '${dirs}',
+          ja: '${dirs}',
+          cn: '${dirs}',
+        },
+      },
+    },
+    {
+      id: 'R1S Headmarker Line Stack',
+      type: 'HeadMarker',
+      netRegex: { id: headMarkerData.lineStack, capture: true },
+      suppressSeconds: 5,
+      response: Responses.stackMarkerOn(),
+    },
+    {
+      id: 'R1S Headmarker Spread Markers',
+      type: 'HeadMarker',
+      netRegex: { id: headMarkerData.spreadMarker, capture: false },
+      suppressSeconds: 5,
+      response: Responses.spread(),
+    },
+    {
+      id: 'R1S Headmarker Pawprint Collector',
+      type: 'HeadMarker',
+      netRegex: { id: headMarkerData.pawprint, capture: true },
+      run: (data, matches) => data.lastPawprintTarget = matches.target,
+    },
+    {
+      id: 'R1S Elevate and Eviscerate Launch Self',
+      type: 'StartsUsing',
+      netRegex: { id: '9446', source: 'Copy Cat', capture: false },
+      condition: (data) => data.me === data.lastPawprintTarget,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Launch Forward (Aim for uncracked tile)',
+        },
+      },
+    },
+    {
+      id: 'R1S Elevate and Eviscerate Launch Other',
```

Comment:
I'm not really sure we need callouts for `Elevate and Eviscerate` targeting other players, it could just be the callouts for when the current player is targeted.

Reply:
It felt important enough to have a trigger for because you have to avoid the `+`-shaped line AoEs that are on them.